e2e/rbd_helper.go

```diff
@@ -630,64 +591,6 @@ func deleteBackingRBDImage(f *framework.Framework, pvc *v1.PersistentVolumeClaim
 	return err
 }
 
-// rbdDuImage contains the disk-usage statistics of an RBD image.
```

Comment:
Can we keep these helpers around, It'll be needed when writing e2e tests for reclaimspace ?

Reply:
sure. let me know which functions you want to keep for reclaimspace?

Reply:
line 633 to 690.
thanks !

Reply:
done